src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -130,7 +130,7 @@ trait MarathonConf
     descr = "Mesos role for this framework. " +
       "If set, Marathon receives resource offers for the specified role in addition to " +
       "resources with the role designation '*'.",
-    default = None)
+    default = Some("*"))
```

Comment:
It is actually correct; I've confirmed with the Mesos team that this is effectively what happens when we omit specifying a role in FrameworkInfo in standard single-role format. We can add it explicitly as "*" in multi-role and it will preserve the same behavior.

I'm inclined to have "*" be the top-level group-role for services not in groups.

Reply:
Ok. Got it.